PWGJE/EMCALJetTasks/AliEmcalJetTask.cxx

```diff
@@ -52,6 +53,8 @@
 #include "AliEmcalParticleJetConstituent.h"
 
 #include "AliEmcalJetTask.h"
+#include "AliPWGJETrainHelpers.h"
```

Comment:
I don't see you using the train helpers - I think this can be removed

Reply:
thanks for spotting, I added this (and the other header) when I was testing different methods and forgot to remove, removed now